CHANGES.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+0.2.5 - (2018-11-21)
```

Comment:
@smpiano change version to `1.0.0` instead of `0.2.5`, this version may not work properly with `airflow` 1.9.0 and we want to make that clear doing the change on the major version. Also add a note on the changelog about this. somthing like `NOTE: This changes where not tested on existing pipelines with 1.9.0`